go/exocom/src/security/manager.go

```diff
@@ -33,6 +34,17 @@ func (m *Manager) ReceiveMessage(message structs.Message) *Result {
 	var messageToSend *structs.Message
 	warningMessage := ""
 	switch message.Name {
+	case "security request":
+		bytes, err := json.Marshal(message.Payload)
+		if err != nil {
+			warningMessage = err.Error()
+		}
+		messageToSend = &structs.Message{}
+		err = json.Unmarshal(bytes, messageToSend)
+		if err != nil {
+			warningMessage = err.Error()
+		}
+		m.authTable[message.ActivityID] = &structs.Message{}
```

Comment:
Can we use a separate table for security requests? I don't like overloading the authTable. Also `message.ActivityID` should already contain something. The flow would be something like this:

* exocom receives "create user" that needs to be authorized
  * exocom sends "authorize message" to security service with activityId 123
* execom receives "security request" with activityId 123
  * execom sends the message in that payload (assume the message in the payload has activityId 456)
* exocom receives message with activityId 456 and isSecurity flag 
  * exocom sends "security response" with activityId 123 and the received message in the payload 
* exocom receives "message authorized" or "message unauthorized" with activityId 123
  * ...